_maps/map_files/generic/CentCom.dmm

```diff
@@ -3968,7 +3968,7 @@
 /turf/open/floor/plasteel/grimy,
 /area/centcom/ferry)
 "om" = (
-/obj/machinery/computer/card/centcom,
```

Comment:
modular pc's dont have the ability to do centcom access im pretty sure

Reply:
if i set `is_centcom = TRUE` they will

Reply:
is_centcom is only partially supported - It can assign centcom jobs but not change centcom access. I would say that the modular computer app for ID consoles is currently not finished.

It also has balances changes from the HTML version, which considers ACCESS_HOP to be the HoP head access allowing them to modify supply jobs. The modular computer app does not and requires ACCESS_CHANGE_IDS to change Supply access.

That being said, the entire thing is also about to be re-written as part of the ID Card design document I'm currently working my way through. This implementation will completely remove the HTML ID console and re-write the modular computer app from the ground up to support the new ID card standards.

Reply:
#54344 This pr is entirely functional only missing map edits and a few minor tweaks

![image](https://user-images.githubusercontent.com/13771064/106959852-de2bb780-6700-11eb-97e4-f5c75983226d.png)
i fixed the centcom thing